apps/avatax/e2e/utils/moneyUtils.ts

```diff
@@ -0,0 +1,24 @@
+import { MoneyFragment } from "../generated/graphql";
+
+const CURRENCY = "USD";
```

Comment:
you can type currency to be currency enum from graphql, so we have safety against typos